frontend/src/ConfirmationPage/ConfirmationPage.jsx

```diff
@@ -183,7 +184,7 @@ class ConfirmationPage extends React.Component {
                   onClick={this.setPassword}
                   disabled={isLoading}
                 >
-                  Finish account setup
+                  Finish account setup{this.props.location.state.inviteToken ? ' and accept invite' : ''}
```

Comment:
Since both mails are sent as an invite, think it's not necessary to add - _**accept invite**_ explicitly. Can keep as - **_Finish account setup_.**